File: app/src/main/java/com/islam/retrofitsoap/webservice/response/response/WeatherResponseBody.java
package com.islam.retrofitsoap.webservice.response.response;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "Body")
public class WeatherResponseBody {
    @Element(name = "getWeatherbyCityNameResponse", required = false)
    public WeatherResponseModel getWeatherbyCityNameResponse;
}

File: app/src/main/java/com/islam/retrofitsoap/webservice/response/tikxmk/WeatherResponseBody.kt
package com.islam.retrofitsoap.webservice.response.tikxmk

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.Xml


@Xml(name = "soap:Body")
class WeatherResponseBody {
    @Element(name = "getWeatherbyCityNameResponse")
    var getWeatherbyCityNameResponse: WeatherResponseModel? =
        null
}